<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MTTControlWidget</class>
 <widget class="QDockWidget" name="MTTControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>145</width>
    <height>199</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(133, 133, 133);</string>
  </property>
  <property name="windowTitle">
   <string>MTT Control Widget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="minimumSize">
    <size>
     <width>145</width>
     <height>177</height>
    </size>
   </property>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>19</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPrev">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNext">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLoadGSM">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load GSM</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
